oauth2/oauth2-core/src/main/java/org/springframework/security/oauth2/core/DelegatingOAuth2TokenValidator.java

```diff
@@ -16,17 +16,19 @@
 
 package org.springframework.security.oauth2.core;
 
+import org.apache.commons.logging.Log;
```

Comment:
It seems there are some unused imports here.

Reply:
Sorry. I'll remove